### PR TITLE
Add WithExposedExceptions engine method #121

### PR DESCRIPTION
**Pull request by BilyachenkoOY**
Pull request for issue #121

### CODE REVIEW

**Comment by BilyachenkoOY** [comment]
Should I add this to `RequestHandlerBuilder`?

File: test/Tests/Adapters/Engine/ExposingExceptionsTests.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        {
+            var engine = GraphQLEngine.New<Query>();
+            var result = await engine
+                .WithExposedExceptions()
+                .NewExecutor()
+                .WithQueryString("query { queryData }")
+                .Execute();
+
+            result.ExposeExceptions.ShouldBeTrue($"{nameof(result.ExposeExceptions)} should be enabled when {nameof(GraphQLEngine.WithExposedExceptions)} is called.");
+        }
```
Comment (tlil) [inline test/Tests/Adapters/Engine/ExposingExceptionsTests.cs:50]: Can you write a test that checks that there's a stacktrace in the output of a failed query as well, please?

**Review by tlil — CHANGES_REQUESTED** [review]
Can I have 1-2 more tests, please? See separate comment.

File: test/Tests/Adapters/Engine/ExposingExceptionsTests.cs (hunk 1 above)
Comment (BilyachenkoOY) [reply test/Tests/Adapters/Engine/ExposingExceptionsTests.cs:50]: Is is enough? Seems that validating output is more about covering GraphQl lib then about GraphQl.Conventions

**Review by tlil — APPROVED** [review]

**Pull request merged**